docs/virtual_machines/network_binding_plugins.md

```diff
@@ -207,3 +207,168 @@ spec:
+can access, or, in the same namespace the VMs reside in.
+
+### Passt sidecar image
+Passt sidecar image is built and pushed to
+[kubevirt quay repository](https://quay.io/repository/kubevirt/network-passt-binding).
+
+The sidecar sources can be found
+[here](https://github.com/kubevirt/kubevirt/tree/release-1.1/cmd/sidecars/network-passt-binding).
+
+The relevant sidecar image needs to be accessible by the cluster and
+specified in the Kubevirt CR when registering the network binding plugin.
+
+### Feature Gate
+If not already set, add the `NetworkBindingPlugins` FG.
+```
+kubectl patch kubevirts -n kubevirt kubevirt --type=json -p='[{"op": "add", "path": "/spec/configuration/developerConfiguration/featureGates/-",   "value": "NetworkBindingPlugins"}]'
+```
+
+> **Note**: The specific passt plugin has no FG by its own. It is up to the cluster
+> admin to decide if the plugin is to be available in the cluster.
+> The passt binding is still in evaluation, use it with care.
+
+### Passt Registration
+As described in the [registration section](#register), passt binding plugin
+configuration needs to be added to the kubevirt CR.
+
+To register the passt binding, patch the kubevirt CR as follows:
+```yaml
+kubectl patch kubevirts -n kubevirt kubevirt --type=json -p='[{"op": "add", "path": "/spec/configuration/network",   "value": {
+            "binding": {
+                "passt": {
+                    "networkAttachmentDefinition": "default/netbindingpasst",
+                    "sidecarImage": "registry:5000/kubevirt/network-passt-binding:devel"
+                }
+            }
+        }}]'
+```
+
+The NetworkAttachmentDefinition and sidecarImage values should correspond with the
+names used in the previous sections, [here](#passt-networkattachmentdefinition)
+and [here](#passt-sidecar-image).
+
+### VM Passt Network Interface
+Set the VM network interface binding name to reference the one defined in the
+kubevirt CR.
+
+```yaml
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-net-binding-passt
+  name: vm-net-binding-passt
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-net-binding-passt
+    spec:
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+          interfaces:
+          - name: passtnet
+            binding:
+              name: passt
+            ports:
+            - name: http
+              port: 80
+              protocol: TCP
+          rng: {}
+        resources:
+          requests:
+            memory: 1024M
+      networks:
+      - name: passtnet
+        pod: {}
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - containerDisk:
+          image: registry:5000/kubevirt/fedora-with-test-tooling-container-disk:devel
```

Comment:
Please don't use `registry:5000`. Maybe `docker.io/kubevirt/fedora-cloud-container-disk-demo:latest`as used in the main interface [doc](https://kubevirt.io/user-guide/virtual_machines/interfaces_and_networks/#start-an-sr-iov-vm).

Reply:
Done